codice/ddf
# DDF-4763 Give Google like results in the results view when a query is fixed by spellchecked

#### What does this PR do?
Gives back feedback and lets users know that their search was spell corrected.

<img width="588" alt="Screen Shot 2019-05-20 at 10 45 54 PM" src="https://user-images.githubusercontent.com/23177404/58069068-1e27c100-7b51-11e9-9c12-2dfb7c0ad6c4.png">
<img width="568" alt="Screen Shot 2019-05-20 at 10 45 59 PM" src="https://user-images.githubusercontent.com/23177404/58069072-2122b180-7b51-11e9-9877-104a8578a4fc.png">
<img width="572" alt="Screen Shot 2019-05-20 at 10 46 06 PM" src="https://user-images.githubusercontent.com/23177404/58069077-23850b80-7b51-11e9-9563-c456895ab627.png">

#### Who is reviewing it? 
@andrewkfiedler 
@tbatie 
@bellcc 
@derekwilhelm 
@willwill96 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@tbatie

#### How should this be tested?
Enable spellcheck in the admin console
Search with single fields and multiple fields with spellcheck on where you misspell your search.
Ensure that results have text that states something like `Showing results for: ...`


#### Any background context you want to provide?
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #____

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
